deps/rabbit/src/rabbit_definitions.erl

```diff
@@ -464,6 +464,12 @@ add_policy(Param, Username) ->
 
 add_policy(VHost, Param, Username) ->
     Key   = maps:get(name,  Param, undefined),
+    case Key == undefined of
```

Comment:
In idiomatic Erlang, you usually match on `undefined`, not compare to it.

Reply:
Thank you for your advise.
Belowing is my modified code
```
    Key   = maps:get(name,  Param, undefined),
    case Key of
      undefined -> exit("policy name not defined");
      _ -> ok
    end,
```

Reply:
You can push to your branch and the PR will be updated.

Reply:
I have push this to my branch. The  PR is updated.